### ISSUE TITLE
Test Reports?

### ISSUE THREAD

**Issue by hughesjs** [gardotd426/regolith-de#7]
Do you want us to let you know what configurations we've tested it on and found it working?

**Comment by gardotd426** [comment]
That would help, yeah.

**Comment by hughesjs** [comment]
I've got the following setup:

Hardware: MS Surface Pro 6
Distro: Manjaro Architect Base with i3 DE.
Kernel: 5.8.3-arch1-1-surface (note the custom kernel)

I just installed over the top of it, even my DM is picking it up properly and letting me switch between them.

As far as I can tell, it all works fine out the box except #6.

**Comment by naib864** [comment]
Hardware: Thinkpad T61
Distro: Manjaro Architect 20.0.3 with XFCE
Kernel:  5.8.1-3-MANJARO

Gnome control panel has problems with bluetooth settings (doesn't work at all) and edge scrolling (touchpad settings), otherwise no issues yet except #6.

**Comment by gardotd426** [comment]
#6 has been fixed... Did you pull the latest commits and reinstall?

**Comment by gardotd426** [comment]
I figured out what's with the @a(ss) nonsense. It's a bug in i3xrocks, it's nothing to do with Arch. 

I found this thread, where people are asking about i3xrocks modules, and keyboard layout was being discussed. kgilmer (the guy behind Regolith Linux) was saying that his was coming back with the exact same @a(ss), it looks like this shouldn't be shipped. I can possibly tweak the command to get it to show `(xkb, us)` (for a US layout, of course) but for now I just think I'm going to remove that block from the equation. I'll work on it later and try and push something. But yeah, just know this isn't an issue with Regolith DE on Arch.

**Comment by naib864** [comment]
> I figured out what's with the @A(ss) nonsense. It's a bug in i3xrocks, it's nothing to do with Arch.

Alright, good to know.

**Comment by naib864** [comment]
Interesting, I just did a fresh install of manjaro-i3 on my T61 and installed regolith and now i3xrocks seems to be completely broken, it doesn't show up at all. Also the theme is not the one I set with regolith-look. Different conf doesn't change that.
It worked on the same machine running manjaro-i3 a few days ago.

![Screenshot from 2020-09-01 01-22-08](https://user-images.githubusercontent.com/66312463/91770391-63a2bd00-ebe1-11ea-8027-ed9587b5a067.png)

**Comment by gardotd426** [comment]
I'll fix it, I just have to remove the block causing it (it's just the keymap block which is pretty useless)

**Comment by pseudofakee** [comment]
Hardware : Asus-G14
Distro: Endeavour OS (Just base packages and core)
Kernel: 5.8.12

Runs pretty well ...better than on Ubuntu ... 

Two main things i've noticed
1) Bluetooth settings is pretty finicky , I had it working once.... but most of the time it doesn't respond
2) On a dual monitor setup , after suspension the workspaces from monitor 2 get thrown onto monitor 1 (primary)

If you need anything , let me know . :) 

Great work :)

**Comment by gardotd426** [comment]
>Runs pretty well ...better than on Ubuntu ...

Thrilled to hear that.

>Bluetooth settings is pretty finicky , I had it working once.... but most of the time it doesn't respond

I'm **pretty** sure that Arch is just finicky with Bluetooth in general. I'll look into it, though. 

>On a dual monitor setup , after suspension the workspaces from monitor 2 get thrown onto monitor 1 (primary)

I'm almost certain this is just an i3 thing. 

Have you saved the i3 config to your local config folder? If not, you need to do that, I think it goes in ~/.config/regolith/i3/config. Once you do that, you need to assign workspaces to a specific monitor. That's just part of i3. After that, suspend shouldn't make a difference.

**Comment by pseudofakee** [comment]
> Thrilled to hear that.

:)

> > Bluetooth settings is pretty finicky , I had it working once.... but most of the time it doesn't respond
> 
Yes, after somemore testing today , it seems no worse than any other time with arch
> 
> 
> I'm almost certain this is just an i3 thing.
> 
> Have you saved the i3 config to your local config folder? If not, you need to do that, I think it goes in ~/.config/regolith/i3/config. Once you do that, you need to assign workspaces to a specific monitor. That's just part of i3. After that, suspend shouldn't make a difference.

Thanks for the tip . I've only ever used i3 on single monitor set ups ... I'm currently using a laptop that I plug in and out of everyday ... so dual displayes isn't static ... Will see what I can mash in the config though .

Once again, great job :)

**Comment by gardotd426** [comment]
>Thanks for the tip . I've only ever used i3 on single monitor set ups ... I'm currently using a laptop that I plug in and out of everyday ... so dual displayes isn't static ... Will see what I can mash in the config though .

Go ahead and set it up as dual-monitor. It will just ignore that when you only have one monitor connected. For example, if you put Workspaces 1-5 on Monitor 1 and 6-10 on Monitor 2, All 10 will go on Monitor 1 until you plug in your second Monitor, then reload i3.

**Comment by pseudofakee** [comment]
Ok . Thank you
Maybe I'm misunderstanding the regolith method . I originally tried to input the workspaces designation through Xresources which didn't seem to work . Doing so in the ./config/regolith/xxxx worked to an extent .... 
What I'm seeing now is that if, for example, I have a window of firefox on workspace 6 (monitor 2) and then lock using mod+ ESC then when I unlock I'm faced with workspace 6 back on monitor 1 .... but with a semi faded 6 sitting in the bar of monitor 2 .

If I try the same with firefox on workspace 7 . Then It just goes back to display 1 , and the faded six is waiting for me on Monitor 2


![Screenshot from 2020-10-03 11-23-34](https://user-images.githubusercontent.com/20177988/94983287-925add00-056b-11eb-90a4-82dad165dd11.png)

Not sure if this is a regolith issue directly , or something specific to this translation ... or even me. 

This is in the i3 config

##Bind workspaces to monitor
workspace "1" output eDP
workspace "2" output eDP
workspace "3" output eDP
workspace "4" output eDP
workspace "5" output eDP
workspace "6" output HDMI-A-0
workspace "7" output HDMI-A-0
workspace "8" output HDMI-A-0
workspace "9" output HDMI-A-0
workspace "10" output HDMI-A-0

So workspaces definitely open on the correct monitor .. But they don't seem to stay there after locking ....

If this is of no relevance to this project , and only reflects my ignorance then I apologise ! :)

**Comment by gardotd426** [comment]
Yeah this is just an i3 thing. Just it mod+shift+r a couple times and it
fixes itself. At least it always does for me (with regular i3 I mean. This
has nothing to do with regolith though).
